### DISCUSSION TITLE
Package group PR is immortal, how to close it?

### DISCUSSION THREAD

**Discussion by puka-tchou** [renovatebot/renovate#8047] [Request Help] [answered]
How can I configure Renovate so it doesn't create immortal PRs when using `packageRules`? I can't figure it out. You'll find my full Renovate configuration below.

```json
{
	"assignees": [
		"gaspacchio"
	],
	"branchPrefix": "dep-",
	"extends": [
		"config:base"
	],
	"labels": [
		"deps"
	],
	"masterIssue": true,
	"prCreation": "immediate",
	"unicodeEmoji": true,
	"vulnerabilityAlerts": {
		"enabled": true,
		"labels": [
			"bug",
			"deps"
		]
	},
	"packageRules": [
		{
			"packagePatterns": [
				"eslint"
			],
			"groupName": "eslint packages"
		},
		{
			"packagePatterns": [
				"^stylelint"
			],
			"groupName": "stylelint packages"
		},
		{
			"packagePatterns": [
				"^(@|)parcel"
			],
			"groupName": "parcel packages"
		},
		{
			"packagePatterns": [
				"^(@|)prettier"
			],
			"groupName": "prettier packages"
		}
	]
}
```

**Comment by rarkins** [comment, marked as answer]
Please provide a simple reproduction repository so that I can suggest alternatives. The problem when you group unrelated packages together but want to ignore them is that there is no common version that can be put in the PR title to uniquely identify that update. And if you close it and it stayed closed, you'd never get future updates. One possibility is to use dependency dashboard approval so that any recreation is stuck behind approval (although so is the original).

**Reply by puka-tchou** [reply]
Thanks for the quick answer. You can find the PR here: https://github.com/pukafever/back-to-the-future/pull/22
I'm sorry, I don't know what is a reproduction repository, could you clarify this?

**Reply by puka-tchou** [reply]
Another way of solving my particular issue would be to ignore the updates that contain a pattern, such as "nightly". Is it possible to ignore specific versions of a package or a group of packages based on a regex expression?

**Reply by viceice** [reply]
https://docs.renovatebot.com/configuration-options/#allowedversions

**Reply by puka-tchou** [reply]
Thanks, @viceice, this solved my issue.
